pom.xml

```diff
@@ -46,24 +36,35 @@
         <developer>
             <name>Chris Cladden</name>
         </developer>
+        <developer>
+            <name>Bryan Turner</name>
+            <email>incommand@hotmail.com</email>
+        </developer>
     </developers>
 
+    <scm>
+        <connection>scm:git:git@github.com:brettwooldridge/NuProcess.git</connection>
+        <developerConnection>scm:git:git@github.com:brettwooldridge/NuProcess.git</developerConnection>
+        <url>git@github.com:brettwooldridge/NuProcess.git</url>
+    </scm>
+    <distributionManagement>
```

Comment:
This is part of the new setup for using `nexus-staging-maven-plugin` to do releases